src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -101,20 +116,65 @@ public class ImportAzureKeyVaultCertificate : KeyVaultCmdletBase
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionWithPolicyParameterSet,
+                    HelpMessage = "Specifies the password for the certificate collection and private key to import.")]
+        [Parameter(Mandatory = false,
+                   ParameterSetName = ImportCertificateFromFileWithPolicyFileParameterSet,
                    HelpMessage = "Specifies the password for the certificate and private key file to import.")]
         [Parameter(Mandatory = false,
-                    ParameterSetName = ImportWithPrivateKeyFromStringParameterSet,
+                    ParameterSetName = ImportWithPrivateKeyFromStringWithPolicyFileParameterSet,
                     HelpMessage = "Specifies the password for the certificate and private key base64 encoded string to import.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionWithPolicyFileParameterSet,
+                    HelpMessage = "Specifies the password for the certificate collection and private key to import.")]
         public SecureString Password { get; set; }
 
+        /// <summary>
+        /// File Path
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   ParameterSetName = ImportCertificateFromFileWithPolicyFileParameterSet,
+                   HelpMessage = "Specifies the path to the file that contains the certificate policy to import to key vault.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromStringWithPolicyFileParameterSet,
+                    HelpMessage = "Specifies the path to the file that contains the certificate policy to import to key vault.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionWithPolicyFileParameterSet,
+                    HelpMessage = "Specifies the path to the file that contains the certificate policy to import to key vault.")]
+        public string PolicyPath { get; set; }
+
+        /// <summary>
+        /// File Path
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   ParameterSetName = ImportCertificateFromFileWithPolicyParameterSet,
+                   HelpMessage = "Specifies the certificate policy to import to key vault.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromStringWithPolicyParameterSet,
+                    HelpMessage = "Specifies the certificate policy to import to key vault.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionWithPolicyParameterSet,
+                    HelpMessage = "Specifies the certificate policy to import to key vault.")]
+        public PSKeyVaultCertificatePolicy Policy { get; set; }
```

Comment:
- Should support piping.
- I prefer to using PolicyObject here to follow https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/piping-best-practices.md